tests/test-coauthors-guest-authors.php

```diff
@@ -2,6 +2,8 @@
 
 class Test_CoAuthors_Guest_Authors extends CoAuthorsPlus_TestCase {
 
+	use Yoast\PHPUnitPolyfills\Polyfills\AssertStringContains;
```

Comment:
This seems irrelevant to the current PR. Did it accidentally sneak in?

Reply:
I added `use Yoast\PHPUnitPolyfills\Polyfills\AssertStringContains;` on purpose. In https://github.com/Automattic/Co-Authors-Plus/pull/774#discussion_r610490537, @GaryJones pointed out using `assertStringContainsString()` over `assertContains()`. `assertContains()` was soft deprecated with PHPUnit 7.5.0, hard depricated with PHPUnit 8.0.0 and removed with PHPUnit 9.0.0. On the other hand, `assertStringContainsString()` had been introduced with PHPUnit 7.5.0. As I'm also running tests against PHP 5.6 with PHPUnit 5.7.27, I need [PHPUnit-Polyfills](https://github.com/Yoast/PHPUnit-Polyfills) to be able to run the unit tests with various PHPUnit versions.